src/Evolution/Systems/CurvedScalarWave/Equations.hpp

```diff
@@ -0,0 +1,110 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GrTags.hpp"
+#include "Utilities/TMPL.hpp"
+
+template <typename>
+class Variables;
+
+class DataVector;
+
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+
+namespace Tags {
+template <typename, typename, typename>
+struct deriv;
+template <typename>
+struct dt;
+template <typename>
+struct NormalDotFlux;
+template <typename>
+struct NormalDotNumericalFlux;
+template <typename>
+struct Variables;
+}  // namespace Tags
+
+namespace CurvedScalarWave {
+struct Psi;
+struct Pi;
+template <size_t Dim>
+struct Phi;
+struct ConstraintGamma1;
+struct ConstraintGamma2;
+}  // namespace CurvedScalarWave
+
+namespace CurvedScalarWave {
+/*!
+ * \brief Compute the time derivative of the evolved variables of the
+ * first-order scalar wave system on a curved background.
+ *
+ * The evolution equations for the first-order scalar wave system are given by
+  * <a href="gr-qc/0407011">
+ * https://arxiv.org/abs/gr-qc/0407011 </a>:
```

Comment:
You have the href and contents swapped.  (It could also be done with markdown syntax.)